technical-committee/shared-development-roadmap/epics/shared/README.md

```diff
@@ -0,0 +1,16 @@
+# CLARIAH Shared Development Roadmap: Shared core Epics
+
+**Shared core epics** describe epic stories providing core shared functionality
```

Comment:
In my view, ‘epics’ is more common than ‘epic stories’ in Agile terminology. Elsewhere you seem to prefer ‘epic’ too.

```suggestion
**Shared core epics** describe epics providing core shared functionality
```